src/Compilers/Core/MSBuildTask/Microsoft.Managed.Core.targets

```diff
@@ -110,9 +110,9 @@
         <MetadataName>%(CompilerVisibleItemMetadata.MetadataName)</MetadataName>
       </_GeneratedEditorConfigMetadata>
 
-      <!-- Record that we'll write a file, and add it to the editorconfig inputs -->
+      <!-- Record that we'll write a file, and add it to the analyzerconfig inputs -->
       <FileWrites Include="$(GeneratedMSBuildEditorConfigFile)" />
-      <EditorConfigFiles Include="$(GeneratedMSBuildEditorConfigFile)" />
+      <AnalyzerConfigFiles Include="$(GeneratedMSBuildEditorConfigFile)" />
```

Comment:
Was this that we made a change to not look at `EditorConfigFiles` and our targets were wrong?

Reply:
Ah, nevermind, I reread the description :)

---
In reply to: [444514499](https://github.com/dotnet/roslyn/pull/45357#discussion_r444514499) [](ancestors = 444514499)